Refined GitHub Extension/styles.js

```diff
@@ -1,13 +1,16 @@
-document.addEventListener('DOMContentLoaded', function() {
-  function addStylesheet(name) {
-    const link = document.createElement("link");
-    link.type = "text/css";
-    link.rel = "stylesheet";
-    link.href = safari.extension.baseURI + name;
-    document.head.appendChild(link);
-  }
-
-  addStylesheet("refined-github.css");
-  addStylesheet("safari.css");
+ function addStylesheet(name) {
+   const link = document.createElement("link");
+   link.type = "text/css";
+   link.rel = "stylesheet";
+   link.href = safari.extension.baseURI + name;
+   document.head.appendChild(link);
+ }
 
-}, false);
+(function inject () {
+  if (document.head) {
+    addStylesheet("refined-github.css");
+    addStylesheet("safari.css");
+  } else {
+    setTimeout(inject, 100);
+  }
+})();
```

Comment:
This could work:

```suggestion
const observer = new MutationObserver(() => {
    if (document.body) {
	    addStylesheet("refined-github.css");
	    addStylesheet("safari.css");
        observer.disconnect();
    }
});
observer.observe(document, {childList: true, subtree: true})
```

I'm waiting for `document.body` because even if `document.head` exists, it doesn't mean that GitHub’s style is already there.